File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        summ = 0
        original_x = x
    # Storing the original value of x
        while x > 0:
            rem = x % 10
            x = x // 10
            summ = summ * 10 + rem
        return summ == original_x


        

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x);
        int l = s.size();
        for(int i = 0;i < l - 1;i++){
            if(s[i] != s[l - i - 1])  return false;
        }
        return true;
    }
};